include/ccf/endpoint_registry.h

```diff
@@ -127,7 +127,7 @@ namespace ccf::endpoints
       std::map<RESTVerb, std::shared_ptr<PathTemplatedEndpoint>>>
       templated_endpoints;
 
-    ccf::Mutex metrics_lock;
+    ccf::Pal::Mutex metrics_lock;
```

Comment:
The ccf coding style is for lower-cased namespaces - can you please rename this to `ccf::pal`?